### PR TITLE
Derived data fix

### PR DESCRIPTION
**Pull request by BykhovDenis**
Fixed multiple update tx, implemented fullRef with _class

### CODE REVIEW

File: plugins/plugin-core-dev/src/connection.ts [hunk 1]
```diff
@@ -107,5 +120,19 @@ class ClientImpl implements WithAccountId {
 }
 
 export async function connect (handler: (tx: Tx) => void): Promise<WithAccountId> {
-  return await ClientImpl.create(handler)
+  const client = await ClientImpl.create(handler)
+  const ddProcessor = await DerivedDataProcessor.create(client.model, client.hierarchy, newClientOnlyStorage(client))
```
Comment (haiodo) [inline plugins/plugin-core-dev/src/connection.ts:124]: It is not required, since it is already in core client.

Comment (BykhovDenis) [reply plugins/plugin-core-dev/src/connection.ts:124]: Required... Because in core client we don't save dd transaction in DB

Comment (haiodo) [reply plugins/plugin-core-dev/src/connection.ts:124]: And we should not save them

Comment (haiodo) [reply plugins/plugin-core-dev/src/connection.ts:124]: We just need notification to update live queries

Comment (haiodo) [reply plugins/plugin-core-dev/src/connection.ts:124]: Ok, since it server simulation we need it.

**Review by haiodo — APPROVED** [review]

**Pull request merged**